frontend/app/package.json

```diff
@@ -91,7 +91,7 @@
     "cross-env": "7.0.3",
     "dotenv": "14.3.0",
     "electron": "18.0.1",
-    "electron-builder": "22.14.13",
+    "electron-builder": "23.0.3",
```

Comment:
OK I am curious, is there a specific reason behind the electron-builder update, and if there is there any reason not to use `23.0.9` instead